### DISCUSSION TITLE
Form asset 413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#5569] [Troubleshooting] [answered]
Is there a way to change the max file size of an asset that can be uploaded to a form? I would also like to display an error message when the file size is exceeded, how would I approach that? Currently any file size larger than 1mb throws a "413 Request Entity Too Large" page.

**Comment by duncanmcclean** [comment, marked as answer]
I believe you can configure this in the `php.ini` file on your server (the max file size), here's a [potential solution](https://stackoverflow.com/a/2184541/3237188). If you use Laravel Forge, you can [edit it from within the UI](https://stackoverflow.com/a/57041507/3237188).

Not sure if the error message is possible, would have to be done client-side as the request wouldn't get to Laravel/Statamic in order to do that kind of validation.

**Reply by fredbouin** [reply]
great with forge that was very simple. I guess for now I'll just put a max file size label instead of the error message.
